pom.xml

```diff
@@ -418,6 +457,158 @@
                 <artifactId>logstash-logback-encoder</artifactId>
                 <version>4.9</version>
             </dependency>
+            <dependency>
```

Comment:
masse nytt eller berre omstrukturering?

Reply:
Noe nytt, men ellers er det bare for å unngå dependency-konflikter.